platform/api/equinixmetal/api.go

```diff
@@ -362,6 +362,7 @@ ExecStart=/usr/bin/curl --retry-delay 1 --retry 120 --retry-connrefused --retry-
 # We don't verify signatures because the iPXE script isn't verified either
 # (and, in fact, is transferred over HTTP)
 
+ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do echo wipefs -af ${disk}; done'
```

Comment:
For this kind of command, we should take the habit to have long options instead of short ones:
```suggestion
ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do echo wipefs --all --force ${disk}; done'
```

What do you think ? It helps for later reviewing - or if you're not familiar with the command. :)